src/core/hle/D3D8/XbPixelShader.cpp

```diff
@@ -1284,7 +1302,7 @@ void DxbxUpdateActivePixelShader() // NOPATCH
   // as these could have been updated via SetRenderState or otherwise :
   D3DXCOLOR fColor[PSH_XBOX_CONSTANT_MAX];
   extern XboxTextureStateConverter NV2ATextureStates;
-  
+  float fogDensity, fogStart, fogEnd, fogTableMode, fogEnable;
```

Comment:
Just a small nitpick: Consider ordering these variables in the same order as they're being referred-to in the code?

Reply:
fixed